aiidalab_widgets_base/computational_resources.py

```diff
@@ -385,19 +392,19 @@ def on_setup_ssh(self, _=None, on_success=None):
             # If hostname & username are not provided - do not do anything.
             if self.hostname.value == "":  # check hostname
                 print("Please specify the computer hostname.")
-                return
+                return False
```

Comment:
See also comment above. I think we should use a `RuntimeError` to indicate the error state.
Add the following exception to the module:
```
class _SshConfigError(RuntimeError):
    pass
```

Reply:
I think raising an exception won't work here as I am also calling the `on_setup_ssh()` method by ipywidgets.

The other possibility would be to have some other function that is raising a `RuntimeError` in case of an issue. I would then call this new function inside `on_setup_ssh()` and handle the exception there.

Maybe, keeping it as-is is still better.

what do you think?

Reply:
I was referring to this:

https://github.com/aiidalab/aiidalab-widgets-base/blob/419449b387b15206e3191f868b1f57098ecf51cf/aiidalab_widgets_base/computational_resources.py#L302-L304

`Setup ssh` button calls this function, so raising an exception in it doesn't look like a good option.